### PR TITLE
new simple  ui test host introduced

### PR DESCRIPTION
**Pull request by tstrych**
============================= test session starts ==============================
collected 37 items / 36 deselected / 1 selected

tests/foreman/ui/test_host.py::test_positive_read_details_page_from_new_ui
================= 1 passed, 36 deselected in 152.00s (0:02:32) =================

### CODE REVIEW

File: tests/foreman/ui/test_host.py
Comment (Griffin-Sullivan) [inline tests/foreman/ui/test_host.py:2369]: So are you waiting for changes before you un-comment this line?

Comment (tstrych) [reply tests/foreman/ui/test_host.py:2369]: Good question, I don't know where to bring this, foreman developers are not creating the HTML structure, they are just using the Patterfly objects, but the structure itself is missing span element, so 3 things in my mind:
1. ask widgetastic developers for help
2. find the workaround in our code ( I was not able to, but maybe you can have an idea :)  ) 
3. this problem is  skipped. Just archived here, in BZ and also in our code,  right now this only occurs for comment, not sure if there can be another field in  `['Overview']['DetailsCard']['details']`

Comment (Griffin-Sullivan) [reply tests/foreman/ui/test_host.py:2369]: I can look into it. Maybe @latran can provide some insight here as well?

Comment (tstrych) [reply tests/foreman/ui/test_host.py:2369]: If you want to, I can give you introduction to the problem :+1: (we can schedule a short call)

Comment (tstrych) [reply tests/foreman/ui/test_host.py:2369]: @ColeHiggins2  I build a reader to read that, it just a mistake that element is missing, right now DEV's looked into that.

File: tests/foreman/ui/test_host.py
Comment (jyejare) [inline tests/foreman/ui/test_host.py:2369]: I think this is a minor assert to hold this PR/test getting merged for! 

I would have used to skip that assert until we have that bug open and get this PR merged:
```
if not bz_bug_is_open(2059298):
    assert values['Overview']['DetailsCard']['details']['comment'] == 'Host with fake data'
```

File: tests/foreman/ui/test_host.py
Comment (jyejare) [inline tests/foreman/ui/test_host.py:2349]: Case importance should be critical for this test?

**Review by jyejare — COMMENTED** [review]
ACK pending comments !

File: tests/foreman/ui/test_host.py
Comment (tstrych) [reply tests/foreman/ui/test_host.py:2369]: it was discussed with developers, and the result is that the problem should be fixed from our side, so something more dynamic was needed.  I was looking for solutions and I came up with locator `.//div[@class="pf-c-description-list__group"]//dd//descendant::*/text()/..` that is looking at all elements that have descendants with text.

File: tests/foreman/ui/test_host.py
Comment (tstrych) [reply tests/foreman/ui/test_host.py:2349]: why isn't the high, high enough?

**Comment by tstrych** [comment]
updated local test results:
```
tests/foreman/ui/test_host.py::test_positive_read_details_page_from_new_ui 
================= 1 passed, 36 deselected in 145.22s (0:02:25) =================
```

**Comment by tstrych** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_positive_read_details_page_from_new_ui'
airgun: 673

**Comment by tstrych** [comment]
PRT error not connected

File: tests/foreman/ui/test_host.py
Comment (JacobCallahan) [reply tests/foreman/ui/test_host.py:2349]: I would leave importance designations up to the component owners.

Comment (jyejare) [reply tests/foreman/ui/test_host.py:2349]: IMHO, reading details from the host details(that too its a new UI) page should be treated as necessary functionality and hence it should be critical.

I will also leave it up to @tstrych to decide, it was just my opinion.

**Review by jyejare — APPROVED** [review]

File: tests/foreman/ui/test_host.py
Comment (tstrych) [reply tests/foreman/ui/test_host.py:2349]: now it's critical, year after it is low, but all tests importances are something that is not changing much even though we as component owners can do that, but it is rock bottom priority for me.

**Review by Gauravtalreja1 — COMMENTED** [review]
@tstrych PRT fails in tests collection could you check?
It seems your robottelo and airgun branches are not rebased on https://github.com/SatelliteQE/robottelo/pull/9390 and https://github.com/SatelliteQE/airgun/pull/672

**Review by ColeHiggins2 — COMMENTED** [review]
Ack pending rebase

**Review by rplevka — APPROVED** [review]

**Comment by tstrych** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_positive_read_details_page_from_new_ui'
airgun: 673

**Comment by omkarkhatavkar** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_positive_read_details_page_from_new_ui'

**Comment by omkarkhatavkar** [comment]
robottelo-runner is passed!

**Pull request merged**
